doc-site/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 4000",
```

Comment:
whats this? o.O

Reply:
run the doc site on a different port (defaults to 3000... same as webpack server)